plugins/copr.py

```diff
@@ -56,6 +60,17 @@ def __init__(self, base, cli):
         if cli is not None:
             cli.register_command(CoprCommand)
 
+    # Useful for forcing a distribution
+    def config(self):
+        cp = self.read_config(self.base.conf, PLUGIN_CONF)
```

Comment:
File "/usr/lib/python3.4/site-packages/dnf-plugins/copr.py", line 65, in config
    cp = self.read_config(self.base.conf, PLUGIN_CONF)
AttributeError: 'Copr' object has no attribute 'base'

Please before you push another commit, do "tito --build --rpm -i; dnf copr enable fooo/bar" and test if it really works.